<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainWindowClass</class>
 <widget class="QMainWindow" name="AppMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual Network</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AppMainWindow.qrc">
    <normaloff>:/AppMainWindow/Resources/VNet_1_0.ico</normaloff>:/AppMainWindow/Resources/VNet_1_0.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetCentral">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="widgetLandingPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_LandingPageInfo">
          <property name="text">
           <string>Open or Create a New Project to start.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCustomize">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionUser_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Manual"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuCustomize"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolbar_main">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_modules">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionModel"/>
   <addaction name="actionCycle_Management"/>
   <addaction name="actionBC_Management"/>
   <addaction name="actionAudit_Management"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Customize">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUser_Preferences"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar_mainWindow"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/File_New.png</normaloff>:/AppMainWindow/Resources/File_New.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/File_Open.png</normaloff>:/AppMainWindow/Resources/File_Open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/File_Save.png</normaloff>:/AppMainWindow/Resources/File_Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/File_SaveAs.png</normaloff>:/AppMainWindow/Resources/File_SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Import_Definition_File">
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/update.png</normaloff>:/AppMainWindow/Resources/update.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Definition File</string>
   </property>
  </action>
  <action name="actionRefresh_from_Ansys">
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/refresh.png</normaloff>:/AppMainWindow/Resources/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh project data</string>
   </property>
  </action>
  <action name="actionThermal_Boundary_Conditions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/TBC.png</normaloff>:/AppMainWindow/Resources/TBC.png</iconset>
   </property>
   <property name="text">
    <string>Boundary Conditions</string>
   </property>
   <property name="toolTip">
    <string>Boundary Conditions</string>
   </property>
  </action>
  <action name="actionBC_Management">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/BC_Management.png</normaloff>:/AppMainWindow/Resources/BC_Management.png</iconset>
   </property>
   <property name="text">
    <string>BC Management</string>
   </property>
   <property name="toolTip">
    <string>BC Management</string>
   </property>
  </action>
  <action name="actionAudit_Management">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/Audit_Management.PNG</normaloff>:/AppMainWindow/Resources/Audit_Management.PNG</iconset>
   </property>
   <property name="text">
    <string>Audit Management</string>
   </property>
   <property name="toolTip">
    <string>Audit Management</string>
   </property>
  </action>
  <action name="actionExternal_BC_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/ETBC.png</normaloff>:/AppMainWindow/Resources/ETBC.png</iconset>
   </property>
   <property name="text">
    <string>External BC Data</string>
   </property>
   <property name="toolTip">
    <string>External Boundary Conditions</string>
   </property>
  </action>
  <action name="actionUser_Preferences">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/User_Preferences.png</normaloff>:/AppMainWindow/Resources/User_Preferences.png</iconset>
   </property>
   <property name="text">
    <string>User Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Set Preferences</string>
   </property>
  </action>
  <action name="actionCustomizeUnits">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/Units.jpg</normaloff>:/AppMainWindow/Resources/Units.jpg</iconset>
   </property>
   <property name="text">
    <string>Customize Units</string>
   </property>
   <property name="toolTip">
    <string>Customize Units</string>
   </property>
  </action>
  <action name="actionCycle_Management">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/CycleSheet_Management.jpg</normaloff>:/AppMainWindow/Resources/CycleSheet_Management.jpg</iconset>
   </property>
   <property name="text">
    <string>Cycle Management</string>
   </property>
   <property name="toolTip">
    <string>Cycle Management</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/File_Close.jpg</normaloff>:/AppMainWindow/Resources/File_Close.jpg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close active project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionHomeDir">
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/HomeDirNew.png</normaloff>:/AppMainWindow/Resources/HomeDirNew.png</iconset>
   </property>
   <property name="text">
    <string>HomeDir</string>
   </property>
   <property name="toolTip">
    <string>Open Home Directory</string>
   </property>
  </action>
  <action name="actionAssemble_Run">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/Assemble_Run_Icon.png</normaloff>:/AppMainWindow/Resources/Assemble_Run_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Assemble &amp; Run</string>
   </property>
   <property name="toolTip">
    <string>Assemble &amp; Run</string>
   </property>
  </action>
  <action name="actionUser_Manual">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/UserManual.png</normaloff>:/AppMainWindow/Resources/UserManual.png</iconset>
   </property>
   <property name="text">
    <string>User Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSolution">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/Solution.PNG</normaloff>:/AppMainWindow/Resources/Solution.PNG</iconset>
   </property>
   <property name="text">
    <string>Solution</string>
   </property>
   <property name="toolTip">
    <string>Solution Module</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Application Details</string>
   </property>
  </action>
  <action name="actionImport_MATROP">
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/ImportImage.png</normaloff>:/AppMainWindow/Resources/ImportImage.png</iconset>
   </property>
   <property name="text">
    <string>Import MATPROP</string>
   </property>
   <property name="toolTip">
    <string>Import Matrop file for Solution Module</string>
   </property>
  </action>
  <action name="actionCompInstruction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/AppMainWindow/Resources/CompInstructions.PNG</normaloff>:/AppMainWindow/Resources/CompInstructions.PNG</iconset>
   </property>
   <property name="text">
    <string>CompInstruction</string>
   </property>
   <property name="toolTip">
    <string>CompInstruction for post processing</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AppMainWindow.qrc">
     <normaloff>:/AppMainWindow/Resources/Model.png</normaloff>:/AppMainWindow/Resources/Model.png</iconset>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
